SPECS-EXTENDED/gspell/gspell.spec

```diff
@@ -67,9 +68,9 @@ find $RPM_BUILD_ROOT -name '*.la' -delete
 
 
 %files -f gspell-1.lang
-%license COPYING
+%license LICENSES/*
 %{_libdir}/girepository-1.0/
-%{_libdir}/libgspell-1.so.2*
+%{_libdir}/libgspell-1.so.3*
```

Comment:
This looks like a breaking change. Did you check other depending on packages whether they're building and running properly?

Reply:
I have grepped the SPECS (`rg Requires:.*gspell`, `rg gspell`) and have not come across a package that depends on `gspell`. It seems nothing depends on it. Please advise if I am mistaken.